www/source/partials/docs/_using-hab-config-workstation.html.md.erb

```diff
@@ -3,9 +3,9 @@ Once Habitat has been installed, the <code>hab</code> CLI makes it easy to get y
 
 <img alt="screenshot of hab setup output in CLI" src="/images/screenshots/hab-setup.png">
 
-Setup asks you to create a new origin and a set of origin keys. Optionally, you can also provide a [GitHub personal access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) to upload packages to the public depot and share them with the Habitat community.
+Setup asks you to create a new origin and a set of origin keys.
 
-> The GitHub personal access token needs information provided from the <code>user:email</code> and <code>read:org</code> OAuth scopes. Habitat uses the information provided through these scopes for authentication and to determine features based on team membership.
+Optionally, you can also provide a Habitat personal access token to upload packages to the public depot and share them with the Habitat community. To obtain a personal access token, [visit your Profile page](<%= builder_web_url %>/#/profile) on the Builder website and follow the instructions; you can apply the token using the `HAB_AUTH_TOKEN` environment variable so that any commands requiring authentication will use it.
```

Comment:
I'd like to simplify this by just linking them to the access token section of the builder guide: https://www.habitat.sh/docs/using-builder/#builder-token